ngraph/frontend/paddlepaddle/src/op/interp.cpp

```diff
@@ -0,0 +1,62 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <ngraph/opsets/opset6.hpp>
+#include "interp.hpp"
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+
+OutputVector nearest_interp_v2 (const NodeContext& node) {
+    auto x = node.get_ng_input("X");
+
+    using InterpolateMode = ngraph::opset6::Interpolate::InterpolateMode;
+    using CoordinateTransformMode = ngraph::opset6::Interpolate::CoordinateTransformMode;
+    using Nearest_mode = ngraph::opset6::Interpolate::NearestMode;
+    using InterpolateAttrs = ngraph::opset6::Interpolate::InterpolateAttrs;
+    using ShapeCalcMode = ngraph::opset6::Interpolate::ShapeCalcMode;
+
+    InterpolateAttrs attrs;
+
+    attrs.mode = InterpolateMode::nearest; //HARDCODE
+
```

Comment:
What does `HARDCODE` mean in this case? This comment can be interpreted that we are doing something bad in the code that requires attention in the future or rework, but actually it is only correct translation of pdpd operation mode to our OpenVINO mode.